### PR TITLE
Productionization Tweaks

### PR DESCRIPTION
**Pull request by colfax23**
- fixed loading states
 - text updates
 - thumbnail update

Builds on #52

### CODE REVIEW

**Comment by remyroy** [comment]
I'll try to test and review this tonight with #52

File: src/react/commands/Eth2Deposit.ts [hunk 1]
```diff
@@ -34,50 +40,53 @@ const requireDepositPackages = (): boolean => {
...
-      return '';
-    }
-  
-    const pythonpath = executeCommandSync("python3 -c \"import sys;print(':'.join(sys.path))\"");
-  
-    const expythonpath = REQUIREMENT_PACKAGES_PATH + ":" + ETH2_DEPOSIT_CLI_PATH + ":" + pythonpath;
-  
-    env.PYTHONPATH = expythonpath;
+const createMnemonic = (language: string): Promise<string> => {
+  return new Promise((resolve, reject) => {
```
Comment (remyroy) [inline src/react/commands/Eth2Deposit.ts:44]: I'm guessing there was a good reason this make `createMnemonic` return a Promise. For UI maybe? Shouldn't we do the same for `generateKeys`?

File: src/react/components/MnemonicGenerationWizard.tsx [hunk 2]
```diff
@@ -124,15 +91,19 @@ const MnemonicGenerationWizard: FC<Props> = (props): ReactElement => {
...
-    } else {
-      props.setMnemonic("one two three four five six seven eight nine ten eleven twelve one two three four five six seven eight nine ten eleven twelve");
-    }
+    console.log("Generating mnemonic...");
+    createMnemonic('english')
+      .then((mnemonic) => {
+        props.setMnemonic(mnemonic);
+      })
+      .catch((error) => {
+        console.log(error);
```
Comment (remyroy) [inline src/react/components/MnemonicGenerationWizard.tsx:105]: Eventually, I think this should be showed to the user somehow. Could be in another PR or tracked in an issue.

**Review by remyroy — COMMENTED** [review]
My whole review. Let me know what you think.

File: package.json [hunk 3]
```diff
@@ -14,18 +14,23 @@
...
     "typescript": "^4.3.5",
     "webpack": "^5.44.0",
     "webpack-cli": "^4.7.2",
     "webpack-serve": "^4.0.0"
   },
   "scripts": {
-    "build": "webpack --config webpack.react.config.js --config webpack.electron.config.js",
-    "build:watch": "webpack --config webpack.react.config.js --config webpack.electron.config.js --watch",
-    "start": "electron ./dist/electron/index.js",
+    "build": "yarn run buildcli && webpack --config webpack.react.config.js --config webpack.electron.config.js",
```
Comment (remyroy) [inline package.json:25]: I'm not sure I want to build the SFE everytime I want to rebuild the react/electron app. Can we have something seperate for those when in dev and maybe something unified for building the final product?

Comment (remyroy) [reply package.json:25]: Unless there is another command I should be using?

**Comment by remyroy** [comment]
I could work on the cross-platform stuff when merging this with #55

**Comment by remyroy** [comment]
Installing on Ubuntu with the deb target show the application named **WagyuKeyGen** in the menu. Can't we put some space in there to show something like **Wagyu Key Gen** instead?
naming-deb

I couldn't find that exact name in the config, but I'm guessing it should be somewhere or it's using some kind of default.

**Comment by remyroy** [comment]
The bundle works fine when installed on my Ubuntu dev machine with the deb target. Great work!

File: package.json [hunk 4]
```diff
@@ -43,5 +48,23 @@
     "style-loader": "^3.0.0",
     "styled-components": "^5.3.0",
     "typeface-roboto": "^1.1.13"
+  },
+  "build": {
+    "appId": "com.electron.wagyukeygen",
```
[the remark this answers is not in the document]
Comment (colfax23) [reply package.json:53]: Done.

File: src/react/commands/Eth2Deposit.ts [hunk 5]
```diff
@@ -14,8 +15,13 @@ const REQUIREMENT_PACKAGES_PATH = "dist/packages";
...
 
-const SFE_PATH = "dist/bin/eth2deposit_proxy";
-const DIST_WORD_LIST_PATH = cwd() + "/dist/word_lists";
+// Path used when testing locally unbundled, calling python SFE
+const SFE_PATH = "build/bin/eth2deposit_proxy";
+const DIST_WORD_LIST_PATH = cwd() + "/build/word_lists";
+
+// Path used when run as an executable clickable bundle, calling python SFE
+const BUNDLED_SFE_PATH = process.resourcesPath + "/../build/bin/eth2deposit_proxy";
+const BUNDLED_DIST_WORD_LIST_PATH = process.resourcesPath + "/../build/word_lists";
```
[the remark this answers is not in the document]
Comment (colfax23) [reply src/react/commands/Eth2Deposit.ts:24]: As far as I know, yes.  Doesn't seem to specify in the docs, but so far things have been working as is.
https://www.electronjs.org/docs/api/process#processresourcespath-readonly

File: src/react/commands/Eth2Deposit.ts (hunk 1 above)
Comment (colfax23) [reply src/react/commands/Eth2Deposit.ts:44]: Yes, just created #61 for it.  I was able to get around reimplementing it for now, but I do think generateKeys should be rewritten the same way.

File: src/react/components/MnemonicGenerationWizard.tsx (hunk 2 above)
Comment (colfax23) [reply src/react/components/MnemonicGenerationWizard.tsx:105]: Great call.  Added #62.

File: package.json (hunk 3 above)
Comment (colfax23) [reply package.json:25]: No, you're right.  Meant to fix this before checking in, consider it done.

**Comment by colfax23** [comment]
I ran into some whitespace path issues when naming the application with spaces.  Probably fixable, but I chose not to focus on it for now.  Created #63
